contracts/smart-account/modules/SessionValidationModules/ISessionValidationModule.sol

```diff
@@ -3,7 +3,10 @@ pragma solidity 0.8.17;
 
 import {UserOperation} from "../BaseAuthorizationModule.sol";
 
-interface ISessionValidationModule {
+abstract contract ISessionValidationModule {
+    bytes4 public constant EXECUTE_SELECTOR = 0xb61d27f6;
```

Comment:
add the comments here that method name is execute() and optimised execute is excute_xxxxx

Reply:
+